examples/qc-checks/check-for-price-outliers/outliers-using-iqr.sql

```diff
@@ -0,0 +1,42 @@
+-- ============================================================
+-- Description:
+-- This query shows you how to check for outliers in a CSV
+-- file on Drive. We define outliers as observations that
+-- fall below Q1 − 1.5 IQR or above Q3 + 1.5 IQR
+-- ============================================================
+
+-- Load data from CSV
+
+@price_ts = use Drive.csv
+--file=/luminesce-examples/price_time_series.csv
+enduse;
+
+-- Calculate the IQRs
+
+@iqr_data = select
+interquartile_range(price) as 'iqr',

```

Comment:
(iqr not used elsewhere)

Reply:
Removed

Reply:
`as "iqr"` `'` are for stirng cliterals `"` for columns